pandas/tests/resample/test_base.py

```diff
@@ -24,6 +24,33 @@
 from pandas.core.indexes.timedeltas import timedelta_range
 from pandas.core.resample import _asfreq_compat
 
+ALL_1D_NO_ARG_INTERPOLATION_METHODS = [
```

Comment:
Can you make this a fixture instead?

Reply:
I made it a fixture, but it cannot be used in parametrized easily in this case. So it becomes a regular input argument with a for loop in the test body. This has a tiny performance advantage, but note that the parameterize version is easier to debug when reading the test result.